libr/parse/filter.c

```diff
@@ -150,12 +150,6 @@ static bool filter(RParse *p, ut64 addr, RFlag *f, RAnalHint *hint, char *data,
 	ut64 off;
 	int bits = p->analb.anal->bits;
 	int seggrn = p->analb.anal->seggrn;
-	#if 1
-	int segoff = 0;
-	#else
-	//TODO: respect segoff
-	int segoff = r_config_get_i (p->analb.anal->coreb.core, "asm.segoff");
-	#endif
```

Comment:
if you arent using it but still want the comment you can do so, the complaint happened because of the `#if 1` part.

```suggestion
#if 0
// TODO: respect segoff
int segoff = r_config_get_i (p->analb.anal->coreb.core, "asm.segoff");
#endif
```